static/schemas/obms/amt-obm-service.json

```diff
@@ -0,0 +1,29 @@
+{
+    "title": "amt-obm-service",
+    "definitions": {
```

Comment:
Hi @yyscamper. Changed descriptions and integers types, as you suggested...

Could you clarify your comment regarding "definitions"? I'm not clear about what you would like to see under "definitions"...